app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughViewModel.kt

```diff
@@ -1,11 +1,21 @@
 package org.oppia.android.app.walkthrough
 
 import androidx.databinding.ObservableField
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import org.oppia.android.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
 /** [ViewModel] for [WalkthroughActivity]. */
 class WalkthroughViewModel @Inject constructor() : ObservableViewModel() {
   val currentProgress = ObservableField(0)
+  val isProgressBarVisible = MutableLiveData<Boolean>(true)
```

Comment:
Default is true, so can we use only 1 helper function to hop between true/false. 
I don't see any use of two helper function here. Or simply pass the param.

Reply:
Done